test/sapphire_devnet.test.ts

```diff
@@ -9,8 +9,10 @@ import { keccak256 } from "../src";
 import TorusUtils from "../src/torus";
 import { generateIdToken, lookupVerifier } from "./helpers";
 
-const TORUS_TEST_EMAIL = "saasas@tr.us";
-const TORUS_IMPORT_EMAIL = "importeduser5@tor.us";
+const TORUS_TEST_EMAIL = "devnettestuser@tor.us";
+const TORUS_HASH_ENABLED_TEST_EMAIL = "saasas@tr.us";
```

Comment:
```suggestion
const TORUS_HASH_ENABLED_TEST_EMAIL = "saasas@tor.us";
```

Reply:
oh this is fine. its just a test case string, if i rectify it now we need to changes the key response comparison in all tests. So its better to keep it like this.